src/main/java/org/jabref/logic/importer/fileformat/MsBibImporter.java

```diff
@@ -18,26 +19,31 @@
 import org.xml.sax.SAXException;
 import org.xml.sax.SAXParseException;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * Importer for the MS Office 2007 XML bibliography format
- * By S. M. Mahbub Murshed
+ * By S. M. Mahbub Murshed & Nicholas S. Weatherley
```

Comment:
Pleas remove this author comment completely as we handle author credits via our [authors-file](https://github.com/JabRef/jabref/blob/master/AUTHORS)